src/components/RelatedAgreementsFieldArray/RelatedAgreementField.js

```diff
@@ -58,7 +58,7 @@ export default class RelatedAgreementField extends React.Component {
         renderTrigger={(props) => (
           <Button
             aria-haspopup="true"
-            buttonRef={this.findAgreementButtonRef}
+            buttonRef={props.buttonRef || this.findAgreementButtonRef}
```

Comment:
Does this now break the behaviour in componentDidMount?

Reply:
Ah, yes it does